src/optimization/ControlFlow.h

```diff
@@ -44,6 +44,11 @@ namespace vc4c
 		 */
 		void addStartStopSegment(const Module& module, Method& method, const Configuration& config);
 
+		/*
+		 * Remove constant load in loops
```

Comment:
Please provide some documentation (see the other optimization-steps), preferable with small example